terraform/cicd-deployment/README.md

```diff
@@ -0,0 +1,442 @@
+| lambda_layer_bucket_prefix | string | "aws-managed-cost-intelligence-dashboards" | Lambda layer S3 bucket prefix |
+| deploy_cudos_dashboard | string | "no" | Deploy CUDOS dashboard |
+| data_buckets_kms_keys_arns | string | "" | KMS key ARNs for data buckets |
+| share_dashboard | string | "yes" | Enable dashboard sharing |
+
+#### Legacy Parameters
+| Variable | Type | Default | Description |
+|----------|------|---------|-------------|
+| keep_legacy_cur_table | string | "no" | Maintain legacy CUR table |
+| cur_bucket_path | string | "s3://cid-{account_id}-shared/cur/" | CUR S3 bucket path |
+| cur_table_name | string | "" | CUR table name |
+| permissions_boundary | string | "" | IAM permissions boundary |
+| role_path | string | "/" | IAM role path |
+
+### Global Values Configuration
+
+| Variable | Type | Default | Validation | Description |
+|----------|------|---------|------------|-------------|
+| destination_account_id | string | null | Must be 12 digits | AWS Account ID for DataExport replication |
+| source_account_ids | string | "" | Must be comma-separated 12-digit IDs | Source AWS account IDs |
+| aws_region | string | "" | - | AWS region for dashboard deployment |
+| quicksight_user | string | null | Required | QuickSight user for dashboard sharing |
+
+
+### Example Usage
+
+### Global Values Configuration
+
+The following values should be configured in your `.tfvars` file as global_values. Only these values need to be configured in the tfvars file, as all other variables are pre-configured with default values in `variables.tf` file
+
+```hcl
+global_values = {
+  destination_account_id = "123456789012"
+  source_account_ids     = "123456789012,987654321098"
+  aws_region             = "us-east-1"
+  quicksight_user        = "user@example.com"
+}
+```
+Each field in global_values represents:
+
+* destination_account_id: The AWS account ID where the dashboard(s) will be deployed
+
+* source_account_ids: Comma-separated list of AWS account IDs that are source/payer/management accounts
+
+* aws_region: The AWS region where resources will be deployed
+
+* quicksight_user: QuickSight user name configured
+
+### Note on Configuration Management:
+By default, only the global values need to be configured in your `.tfvars` file, as all other variables are pre-configured with default values in `variables.tf`. However, we strongly recommend:
+
+* Reviewing all default values in `variables.tf` to ensure they meet your requirements
+
+* Either modifying the default values directly in `variables.tf`, or
+
+* Adding additional variable declarations in your `.tfvars` file to override specific defaults as needed
+
+## Validation Rules
+- Destination account ID must be exactly 12 digits
+- Source account IDs must be comma-separated 12-digit numbers
+- QuickSight user is required and cannot be null
+- Most boolean flags use "yes"/"no" string values
+
+## CloudFormation Stack Outputs
```

Comment:
Do we need this?

Reply:
this looks like repetition of CFN code, not sure if this all section brings any value to cx

Reply:
and just increasing repetitions to manage